src/index/Index.cpp

```diff
@@ -922,9 +922,9 @@ void Index::createFromOnDiskIndex(const string& onDiskBase,
             << std::endl;
   if (allPermutations) {
     _opsMeta.setup(onDiskBase + ".index.ops" + MMAP_FILE_SUFFIX,
-                   ad_utility::ReuseTag());
+                   ad_utility::ReuseTag(), ad_utility::AccessPattern::Random);
```

Comment:
Am I seeing this right, `ReuseTag` needs to be a tag to select at compile time but `AccessPattern` can be an enum because it selects at runtime?

Reply:
If you want to I can also make ReuseTag an enum. It is basically the same as in an fstream where we also can decide by a runtime argument wheter we want to overwrite an existing file or use/add to its contents. The efficiency difference would not matter and maybe it would be more consistent. Just tell me, what you think.